templates/AMI-Template-Instances.cfn.yml.j2

```diff
@@ -48,19 +82,29 @@ Resources:
           Ebs:
             Encrypted: true
             DeleteOnTermination: true
-            VolumeSize: {{ disk_size }}
+            VolumeSize: !Ref {{ layer }}EC2SecondaryDiskSize
             VolumeType: "gp2"
       IamInstanceProfile:
-        Fn::ImportValue: !Sub "${Environment}${ApplicationName}{{ instance_profile_prefix | default('') }}InstanceProfileName"
-      # Amazon Linux 2023: al2023-ami-2023.4.20240611.0-kernel-6.1-x86_64
+        Fn::ImportValue:
```

Comment:
sub is nicer to read then join.

Reply:
Yeah, I'd like to, but that would mean wrapping `{{ }}` inside `${ }` and I decided using `Join` was simpler than adding `{% raw %} {% endraw %}` tags.